src/components/JobContent.astro

```diff
@@ -9,44 +9,6 @@
   width: 100%;
   max-width: 680px;
   margin: 5.8rem auto 0;
-
-  h3,
-  h4 {
-    color: $dark-green;
-  }
-
-  h3 {
-    font-size: 28px;
-    margin-bottom: 1rem;
-  }
-
-  h4 {
-    font-size: 20px;
-    margin: 2rem 0 1rem;
-  }
-
-  p {
-    margin-bottom: 1rem;
-    color: $grey;
-  }
-
-  a {
-    color: $blue;
-
-    &:hover {
-      text-decoration: underline;
-    }
-  }
-
-  ul {
-    list-style-type: disc;
-    margin-left: 1.5rem;
-    margin-bottom: 1.5rem;
-
-    li {
-      color: $grey;
-      margin-bottom: 0.5rem;
-    }
-  }
+  @include markdown-style;
```

Comment:
`JobContent.astro` is currently the only place where the markdown is used. If we added it to elsewhere in the future too, we can either just import this scss mixin or we could move this style to a global scss(if there are many pages).